src/TourManager/TourManager.Service.Implementation/TourManager.Service.Implementation.csproj

```diff
@@ -4,6 +4,10 @@
         <TargetFramework>net5.0</TargetFramework>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="FluentValidation.AspNetCore" Version="9.5.0" />
```

Comment:
This is a not Asp.net core project, please remove this reference and add just FluentValidation package

Reply:
okay